resources/resources.go

```diff
@@ -624,28 +624,29 @@ type Message struct {
 	ArrivedAt          *RFC3339DateTime `json:",omitempty"`
 	AttachmentCount    int              `json:",omitempty"`
 	AttemptCount       int              `json:",omitempty"`
-	CampaignID         int64            `json:",omitempty"`
 	CampaignALT        string           `json:",omitempty"`
-	ContactID          int64            `json:",omitempty"`
+	CampaignID         int64            `json:",omitempty"`
 	ContactALT         string           `json:",omitempty"`
+	ContactID          int64            `json:",omitempty"`
+	CustomID           string           `json:",omitempty"`
 	Delay              float64          `json:",omitempty"`
-	Destination        Destination
```

Comment:
Hey. This is breaking change, please add `// Deprecated: use DestinationID field instead.` comment